js/tokocrypto.js

```diff
@@ -1607,7 +1608,10 @@ module.exports = class tokocrypto extends Exchange {
                     request['quoteOrderQty'] = this.decimalToPrecision (quoteOrderQty, TRUNCATE, precision, this.precisionMode);
                     params = this.omit (params, [ 'quoteOrderQty', 'cost' ]);
                 } else if (price !== undefined) {
-                    request['quoteOrderQty'] = this.decimalToPrecision (amount * price, TRUNCATE, precision, this.precisionMode);
+                    const amountString = amount.toString ();
```

Comment:
We can't use `toString()` here for reasons explained in the CONTRIBUTING doc: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

<img width="1484" alt="Screenshot 2023-02-07 at 19 44 40" src="https://user-images.githubusercontent.com/1294454/217323743-b1c3a212-944d-44c8-91de-7de46506689e.png">

Let's replace it with `this.numberToString (...)`.